2. transformation/houses.py

```diff
@@ -0,0 +1,49 @@
+""" 
+    A script that reads in from the locational data pickle and outputs a pickle 
+    of a housing dataframe from DEFAULT_GEOCODE for first iteration
+"""
+import pandas as pd
+import pickle
+import os
+
+# Accessing the DataFiles Path
+home_dir = os.path.expanduser("~")
+datafiles_path = os.path.join(home_dir, "Desktop", "DataFiles")
+
+# Setting the FilesOut and the AppStaging for the pickles
+pickle_file_path = os.path.join(datafiles_path, "AppStaging", 'locational-data.pkl')
+files_out_path = os.path.join(datafiles_path, "FilesOut")
+output_pickle_path = os.path.join(datafiles_path, "AppStaging", 'housing_data.pkl')
```

Comment:
same comments for global variables as above

Reply:
Happy to make the global variables a config going forward, but for this it is a very simple script that will not change or increase in complexity.